.gitignore

```diff
@@ -36,3 +36,9 @@ operator/tests/e2e-v2-helm/
 
 # Go releaser artifacts
 dist/
+
+# Dep Chart Lock file
+Chart.lock
```

Comment:
Didn't we determine that we need `Chart.lock` to be checked in? https://github.com/redpanda-data/helm-charts/blob/main/charts/redpanda/Chart.lock

Reply:
Good question. I'm going back and fourth with this. Our tests in redpanda repo requires Chart.lock to have stable input. I didn't want operator helm chart lock as I feel like it's not required. Never the less if .helmignore will exclude Chart.lock I can commit `Chart.lock`.

Reply:
We don't currently have `Chart.lock` in our `.helmignore` either. That also caused problems.